### DISCUSSION TITLE
Jump host help

### DISCUSSION THREAD

**Discussion by mg-dev25** [isontheline/pro.webssh.net#773] [Q&A] [answered]
Hello, this app look great but before buying it, I need to know if I can use Jump host feature.

I have a server YunoHost, which can be connected from everywhere, and I want to connect to my local machine which can't be connected outside the local network.  I don't know how to do it with the IOS app? Can you help me? Thanks 

The server are in Debian 11 and the second host are in arch Linux. The two host are not default SSH port and use private key for connection.

**Comment by isontheline** [comment]
Hello @mg-dev25 👋

Thanks for considering WebSSH 👍

Jump host feature is not yet available : https://github.com/isontheline/pro.webssh.net/issues/199

But! 😁

You can however use Dynamic Port Forwarding to achieve your needs : 

https://webssh.net/documentation/help/networking/port-forwarding/

**Reply by mg-dev25** [reply]
Do I need to modify my rooter firmwall or my ssh configuration? 

For exemple (all is fake):

host1 Yunohost : 
user: rom
domain (host): kamir
port: 5534

host 2 (local):
user: kevin
domain (host): 192.168.10.100
port: 6633

The private key should be the one for host 1?
What do I need to enter in port forwarding?

thanks

**Reply by isontheline** [reply]
Create a tunnel with :
host1 Yunohost :
user: rom
domain (host): kamir
port: 5534
Private key or password : for host1
Port forwarding (wildcard char) : *

Create a SSH connection :
host 2 (local):
user: kevin
domain (host): 192.168.10.100
port: 6633

Private key or password : for host2


1. Now launch tunnel (host1)
2. Next launch your SSH (host2) connection

So you will be able to access host2 through host1

Could you try?

**Comment by mg-dev25** [comment, marked as answer]
So, I try and it work  great, I will buy it for my iOS devices. Do you know with this setting how to use it with the ssh -J option in Android Termux terminal? Or other option? Thanks

**Reply by isontheline** [reply]
You may ask to the dev as I don't own an Android device to help you

**Reply by mg-dev25** [reply]
Ok, I will test some syntaxes. Termux is a Linux version on debian, so any Linux command work.
